### ISSUE TITLE
[HOLD for payment 2022-11-18] [$250] Blue border around `Confirm` Button has extra space on Password Protected Pdf Modal reported by @huzaifa-99

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#12520]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open a password protected pdf (either from an existing chat or send a new one in a chat).
2. A modal with open, click on 'enter the password'
3. Press Tab and try to focus on 'Confirm' button.
4. Notice the blue focus outline with extra padding/space on top

## Expected Result: 
The focus outline should have no extra padding/space on top

## Actual Result: 
Focus outline has extra padding/space on top

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- Web

**Version Number:** 1.2.24-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/200353608-941e91b8-4420-4efe-b788-0a199c6b7c9e.mp4

https://user-images.githubusercontent.com/43996225/200353738-40f55f5a-aaf6-43a5-92ad-55ec6f972d77.mp4

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @huzaifa-99
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1667820428100109

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Puneet-here** [comment]
### Proposal 

Use `mt4` instead of `pt4`
https://github.com/Expensify/App/blob/0298f24f5b2c1930de33baf05a767cc5986f3f72/src/components/PDFView/PDFPasswordForm.js#L142

**Comment by kevinksullivan** [comment]
https://www.upwork.com/jobs/~0115543468fa3aedee

@huzaifa-99 please apply for reporting!

**Comment by parasharrajat** [comment]
Let's hold this issue.. We are creating a follow up PR for https://github.com/Expensify/App/issues/11908 which might fix this indirectly. 

cc: @kevinksullivan

**Comment by huzaifa-99** [comment]
> https://www.upwork.com/jobs/~0115543468fa3aedee
> 
> @huzaifa-99 please apply for reporting!

Applied, thanks

**Comment by markojovanovic410** [comment]
We check first html element with chrome developer tools first.
And then we confirm which script or style make the space. 
remove or change style ("user-select: none" or something).

**Comment by mountiny** [comment]
@kevinksullivan So a contributor is fixing this issue as well in this PR https://github.com/Expensify/App/pull/12547 since it is touching the flow and it kind of overlapped with another issue they worked on (this PR is a follow up to existing one).

Since the solution is exactly the same as @Puneet-here proposal [here](https://github.com/Expensify/App/issues/12520#issuecomment-1305859547), I think it is fair to pay them $250 and also the reporting bonus is being paid to @huzaifa-99.

I have linked that PR and this issue should be paid out once that is merged and passes the one week regression period.

**Comment by Puneet-here** [comment]
Thanks for the consideration @mountiny.

**Comment by dylanexpensify** [comment]
all paid out and contracts ended!
